### ISSUE TITLE
Dynamically load newest model prices

### ISSUE THREAD

**Issue by silver5753** [AgentOps-AI/tokencost#22]
https://github.com/AgentOps-AI/tokencost/blob/0eb8d23999bc12670643a89af2cf4be64747f668/tokencost/constants.py#L25

Python packages have a tendency to never get updated, so it might be a good idea to add a call to the github repo's raw yaml when loading the package. That way at least the prices stay up to date.

Something like this:
```
import requests
response = requests.get('https://raw.githubusercontent.com/AgentOps-AI/tokencost/main/tokencost/model_prices.yaml')
TOKEN_COSTS = yaml.safe_load(x.text)
```

**Comment by areibman** [comment]
Good idea.

There is a chance that we may not update the prices ASAP. As such, we should probably switch our price model to a repo that more actively keeps these prices updated. We grabbed our most recent prices from [LiteLLM]. (https://github.com/BerriAI/litellm/blob/main/model_prices_and_context_window.json). I've also included a fallback file `model_prices.json` in case the request doesn't go through.

I'm raising a pull request and major version update to include this #24
